### PR TITLE
[CONTENT] Add translations to SRP Reveal feature

### PR DESCRIPTION
**Pull request by gantunesr**
**Description**

Add translations to SRP Reveal feature

**Screenshots/Recordings**

~_If applicable, add screenshots and/or recordings to visualize the before and after of your change_~

**Issue**

Progresses https://github.com/MetaMask/mobile-planning/issues/583

**Checklist**

* [x] There is a related GitHub issue
* [ ] Tests are included if applicable
* [ ] Any added code is fully documented

### CODE REVIEW

**Review by wachunei — APPROVED** [review]
LGTM

File: locales/languages/fr.json [hunk 1]
```diff
@@ -2189,5 +2189,32 @@
     "description": "Votre adresse IP peut être divulguée sur les serveurs de GitHub si vous activez la recherche automatique de mises à jour. On pourra seulement déterminer que votre adresse IP utilise MetaMask. Aucune autre information ou adresse de compte ne sera divulguée.",
     "primary_action": "Activer l’identification automatique de failles de sécurité",
     "secondary_action": "Non merci"
+  },
+  "srp_security_quiz": {
+    "title": "Quiz sur la sécurité",
+    "introduction": "Pour révéler votre Phrase secrète de récupération, vous devez répondre correctement à deux questions",
```
Comment (NicolasMassart) [inline locales/languages/fr.json:2195]:
```suggestion
    "introduction": "Pour révéler votre phrase secrète de récupération, vous devez répondre correctement à deux questions",
```

File: locales/languages/fr.json [hunk 2]
```diff
@@ -2189,5 +2189,32 @@
...
+    "get_started": "Commencer",
+    "learn_more": "En savoir plus",
+    "try_again": "Réessayer",
+    "continue": "Continuer",
+    "of": "sur",
+    "question_one": {
+      "question": "Si vous perdez votre Phrase secrète de récupération, MetaMask...",
```
Comment (NicolasMassart) [inline locales/languages/fr.json:2202]:
```suggestion
      "question": "Si vous perdez votre phrase secrète de récupération, MetaMask...",
```

File: locales/languages/fr.json [hunk 3]
```diff
@@ -2189,5 +2189,32 @@
...
+      "right_answer": "Ne pourra pas vous aider",
+      "wrong_answer": "Pourra la récupérer pour vous",
+      "right_answer_title": "En effet ! Personne ne peut vous aider à récupérer votre Phrase secrète de récupération.",
```
Comment (NicolasMassart) [inline locales/languages/fr.json:2205]:
```suggestion
      "right_answer_title": "En effet ! Personne ne peut vous aider à récupérer votre phrase secrète de récupération.",
```

File: locales/languages/fr.json [hunk 4]
```diff
@@ -2189,5 +2189,32 @@
...
+      "right_answer_description": "Gravez-la sur une plaque en métal ou inscrivez-la sur plusieurs bouts de papier et cachez-les dans différents endroits secrets pour ne jamais la perdre. Si vous la perdez, il n'y a aucun moyen de la récupérer.",
+      "wrong_answer_title": "C'est faux ! Personne ne peut vous aider à récupérer votre Phrase secrète de récupération.",
```
Comment (NicolasMassart) [inline locales/languages/fr.json:2207]:
```suggestion
      "wrong_answer_title": "C'est faux ! Personne ne peut vous aider à récupérer votre phrase secrète de récupération.",
```

File: locales/languages/fr.json [hunk 5]
```diff
@@ -2189,5 +2189,32 @@
...
+      "wrong_answer_description": "Personne ne peut vous aider à récupérer votre Phrase secrète de récupération si jamais vous la perdez. "
```
Comment (NicolasMassart) [inline locales/languages/fr.json:2208]:
```suggestion
      "wrong_answer_description": "Personne ne peut vous aider à récupérer votre phrase secrète de récupération si jamais vous la perdez. "
```

File: locales/languages/fr.json [hunk 6]
```diff
@@ -2189,5 +2189,32 @@
...
+    },
+    "question_two": {
+      "question": "Si un membre du service d'assistance ou toute autre personne vous demande votre Phrase secrète de récupération...",
```
Comment (NicolasMassart) [inline locales/languages/fr.json:2211]:
```suggestion
      "question": "Si un membre du service d'assistance ou toute autre personne vous demande votre phrase secrète de récupération...",
```

File: locales/languages/fr.json [hunk 7]
```diff
@@ -2189,5 +2189,32 @@
...
+      "right_answer": "Ne la lui fournissez pas, car cette personne essaie de vous arnaquer.",
+      "right_answer_title": "C'est exact ! Vous ne devez jamais partager votre Phrase secrète de récupération avec qui que ce soit.",
```
Comment (NicolasMassart) [inline locales/languages/fr.json:2213]:
```suggestion
      "right_answer_title": "C'est exact ! Vous ne devez jamais partager votre phrase secrète de récupération avec qui que ce soit.",
```

File: locales/languages/fr.json [hunk 8]
```diff
@@ -2189,5 +2189,32 @@
...
+      "right_answer_description": "Toute personne qui vous demande votre Phrase secrète de récupération, que ce soit pour des raisons de sécurité ou autre, essaie de vous arnaquer. ",
```
Comment (NicolasMassart) [inline locales/languages/fr.json:2214]:
```suggestion
      "right_answer_description": "Toute personne qui vous demande votre phrase secrète de récupération, que ce soit pour des raisons de sécurité ou autre, essaie de vous arnaquer. ",
```

File: locales/languages/fr.json [hunk 9]
```diff
@@ -2189,5 +2189,32 @@
...
+      "wrong_answer": "Vous devez la lui fournir",
+      "wrong_answer_title": "C'est faux ! Vous ne devez jamais partager votre Phrase secrète de récupération avec qui que ce soit.",
```
Comment (NicolasMassart) [inline locales/languages/fr.json:2216]:
```suggestion
      "wrong_answer_title": "C'est faux ! Vous ne devez jamais partager votre phrase secrète de récupération avec qui que ce soit.",
```

File: locales/languages/fr.json [hunk 10]
```diff
@@ -2189,5 +2189,32 @@
...
+      "wrong_answer_description": "Toute personne qui vous demande votre Phrase secrète de récupération, que ce soit pour des raisons de sécurité ou autre, essaie de vous arnaquer. "
```
Comment (NicolasMassart) [inline locales/languages/fr.json:2217]:
```suggestion
      "wrong_answer_description": "Toute personne qui vous demande votre phrase secrète de récupération, que ce soit pour des raisons de sécurité ou autre, essaie de vous arnaquer. "
```

**Review by NicolasMassart — COMMENTED** [review]
NIT: Common names do not take uppercase initial in french.

**Pull request merged**